chain/network/src/routing_table_actor.rs

```diff
@@ -68,6 +68,10 @@ pub enum RoutingTableMessages {
         peer_id: PeerId,
         ibf_msg: RoutingVersion2,
     },
+    #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+    StartRoutingTableSync {
```

Comment:
let's start improving the documentation:

can you add the description of what will happen when this message is received?

Reply:
Yes, once we merge other PR changes I'll add documentation.